### ISSUE TITLE
PVp plugin

### ISSUE THREAD

**Issue by Nitacraft** [Lukas0610/PvP2.0#5]
Hello there, I have a question. You change the plugin? It work's with arena? Because I create arena and now doesnt work. One more thing. I need stats for the best 10 players (kills) Sorry for my english. Please response quicly because I have event on monday and I try to find a plugin!

**Comment by Lukas0610** [comment]
Hey,

if you cannot create a arena, there should be a Error-Message in the chat or the log.
It would help me if you could paste this here

**Comment by Nitacraft** [comment]
No, you change completely the plugin and you dont write the commands.. and the /help command dosnt say how I can create arena!

**Comment by Lukas0610** [comment]
All commands are listed in the Wiki if you mean this: https://github.com/Lukas0610/PvP2.0/wiki
How to create an arena: https://github.com/Lukas0610/PvP2.0/wiki/Create-an-arena
